packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -290,10 +275,6 @@ export class FASTSlider extends FormAssociatedSlider implements SliderConfigurat
     }
 
     protected keypressHandler = (e: KeyboardEvent) => {
-        if (this.readOnly) {
```

Comment:
Should this instead be a check for `if (this.disabled)` like we have in the mouse event handlers?

Reply:
AFAICT, the disabled state prevents focus so keyboard events shouldn't register.

Reply:
Great! I thought that might be the case but wasn't sure.